docs/pages/management/guides/joining-nodes-aws-ec2.mdx

```diff
@@ -46,48 +46,7 @@ more in the following guide:
 
 ## Prerequisites
 
-<Tabs>
-<TabItem scope={["oss"]} label="Self-Hosted">
-
-- A running Teleport cluster. For details on how to set this up, see [Getting
-  Started on a Linux Server](../../try-out-teleport/linux-server.mdx).
-
-- The `tctl` admin tool version >= (=teleport.version=).
-
-  ```code
-  $ tctl version
-  # Teleport v(=teleport.version=) go(=teleport.golang=)
-  ```
-
-  See [Installation](../../installation.mdx) for details.
-
-- An AWS EC2 instance to act as a Teleport Node, with the Teleport binary
-  installed. The Node should not have an existing data dir (`/var/lib/teleport` by default).
-  Remove the data directory if this instance has previously joined a Teleport cluster.
-
-</TabItem>
-<TabItem
-  scope={["enterprise"]} label="Enterprise">
-
-- A running Teleport cluster. For details on setting this up, see our
-  [Enterprise getting started guide](../../choose-an-edition/teleport-enterprise/getting-started.mdx).
-
-- The `tctl` admin tool version >= (=teleport.version=), which you can download
-  by visiting the
-  [customer portal](https://dashboard.gravitational.com/web/login).
-
-  ```code
-  $ tctl version
-  # Teleport v(=teleport.version=) go(=teleport.golang=)
-  ```
-
-- An AWS EC2 instance to act as a Teleport Node, with the Teleport binary
-    installed. The Node should not have an existing data dir
-    (`/var/lib/teleport` by default). Remove the data directory if this
-    instance has previously joined a Teleport cluster.
-
-</TabItem>
-</Tabs>
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
```

Comment:
I think we should include the information in this Prerequisites section but not in `edition-prereqs-tabs.mdx` below the partial.

Reply:
Good catch!